homework-2/utube/src/components/video_preview.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+export class VideoPreview extends Component {
+    render () {
+        const preview = this.props.preview;
+        return (
+            <li className="list-group-item" onClick={() => this.props.activatePreview(preview)}>
```

Comment:
it's better to create an instance method which would be responsible for click instead of doing it inside JSX.

You current JSX would create new function every time if components renders